src/Access/Dialog.vala

```diff
@@ -54,60 +54,56 @@ public class Access.Dialog : Granite.MessageDialog {
     }
 
     construct {
-        skip_taskbar_hint = true;
+        unowned var surface = get_surface ();
+        if (surface is Gdk.X11.Surface) {
+            unowned var x11_surface = (Gdk.X11.Surface) surface;
+            x11_surface.set_skip_taskbar_hint (true);
+        }
+
```

Comment:
We don't have an surface until `realize()`, this needs to be done in the `realize.connect()`.

Reply:
Good catch! Fixed it.